packages/node/src/node-fs.ts

```diff
@@ -1,13 +1,13 @@
-import fs from 'fs';
-import path from 'path';
-import { chdir, cwd } from 'process';
-import { promisify } from 'util';
+import fs from 'node:fs';
+import path from 'node:path';
+import { chdir, cwd } from 'node:process';
+import { promisify } from 'node:util';
 
-import { createFileSystem } from '@file-services/utils';
 import type { IBaseFileSystem, IFileSystem, IFileSystemPath } from '@file-services/types';
-import { NodeWatchService, INodeWatchServiceOptions } from './watch-service';
+import { createFileSystem } from '@file-services/utils';
+import { INodeWatchServiceOptions, NodeWatchService } from './watch-service';
 
-const caseSensitive = !fs.existsSync(__filename.toUpperCase());
+const caseSensitive = !fs.existsSync(cwd().toUpperCase());
```

Comment:
`cwd()` could easily be `/` or `C:\`

Reply:
Also, the directory at `cwd()` doesn't necessarily exist.

Reply:
True. How about `process.argv[0]`? I think that would work for this check... and would work for both the esm and cjs targets...